<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>공통 다이얼로그 사용하기</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_file_dialog">
     <property name="text">
      <string>QFileDialog</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_font_dialog">
     <property name="text">
      <string>QFontDialog</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_color">
       <property name="text">
        <string>Text Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_background">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_str_input">
       <property name="text">
        <string>String Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_int_input">
       <property name="text">
        <string>Int Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_item_input">
       <property name="text">
        <string>Item Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_font">
     <property name="text">
      <string>Some Text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
